docs/recipes/debugging-with-webstorm.md

```diff
@@ -0,0 +1,19 @@
+# Debugging AVA tests with WebStorm
+
+Starting with version 2016.2, [WebStorm](https://www.jetbrains.com/webstorm/) and other JetBrains IDEs (IntelliJ IDEA Ultimate, PhpStorm, PyCharm Professional, and RubyMine with installed Node.js plugin) allow you to debug AVA tests. 
+
+## Setup
+
+Add a new *Node.js Run/Debug configuration*: select `Edit Configurations...` from the drop-down list on the top right, then click `+` and select *Node.js*.
+
+In the `JavaScript file` field specify the path to AVA in the project's `node_modules` folder: `node_modules/.bin/ava` on OS X and Linux or `node_modules/.bin/ava.cmd` on Windows.
```

Comment:
@develar Unrelated, but would be nice if the user could just specify the binary name, like `ava`, instead of the full path, like they can in `npm run script`. Pretty easy to do. You can see how it's done here: https://github.com/sindresorhus/npm-run-path/blob/master/index.js